Dispatcher/Supporting Files/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Poppins-Regular.ttf</string>
```

Comment:
If you use only 2 types of fonts - delete the other unused .ttf files.
Each file we add to our project makes the app heavier, and we want our app to be as light as possible